src/main/scala/org/broadinstitute/dsde/workbench/sam/google/DisableUsersMonitor.scala

```diff
@@ -164,12 +165,13 @@ class DisableUsersMonitorActor(
       acknowledgeMessage(ackId).map(_ => StartMonitorPass)
     }
 
-  private def attemptToDisableUser(message: PubSubMessage) = {
+  private def attemptToDisableUser(message: PubSubMessage): Future[Product] = {
     logger.debug(s"received disable user message: $message")
     val userId = WorkbenchUserId(message.contents)
     Tracing.trace("DisableUsersMonitor-PubSubMessage") { span =>
       userService
         .disableUser(userId, samRequestContext = SamRequestContext(Option(span)))
+        .unsafeToFuture()
```

Comment:
might as well just change it to IO?

Reply:
I tried and it was a bit of a rabbit hole, I'd like to leave that for another pr.